### ISSUE TITLE
Invalid gene search 

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#2587]
**Describe the bug**
A gene search is returning the following message - Invalid search hash: 9c1dad4efb1685e85965fb017bd53132

**Link to page(s) where bug is occurring**
Variant page - https://seqr.broadinstitute.org/variant_search/results/39b7adc3d754e3b4dc5e23dc20cab048?page=1&sort=in_omim
Error page - https://seqr.broadinstitute.org/variant_search/results/9c1dad4efb1685e85965fb017bd53132?page=1&sort=in_omim

**Scope of the bug**
Not sure

**Screenshots**
<img width="1255" alt="Screen Shot 2022-03-25 at 10 31 16 AM" src="https://user-images.githubusercontent.com/37872186/160140820-8b437b41-433e-4905-b1f7-7183026bb523.png">

**Comment by lynnpais** [comment]
Yes, I clicked 'Gene Search' link to identify other variants in the same gene. I recently found that the 'seqr' link to identify other cases in the callset with the same variant, is returning the same error message.

**Comment by hanars** [comment]
when I click "Gene search" it works fine and take me to this page: https://seqr.broadinstitute.org/variant_search/results/734e8c66c224d62c627779b065b679d1?page=1&sort=in_omim

The more details you provide the more likely I am to be able to help figure out what happened for you.  Which variant and gene, for example, are really important pieces of information. Have you tried this on other searches/ other projects? Did you get the same issue or not?

**Comment by lynnpais** [comment]
I tried the 'Gene Search' link under the gene for All variants here -https://seqr.broadinstitute.org/project/R0316_cmg_kang_genomes/saved_variants/family/F016086_kang1181/Review

Similarly, I tried the 'seqr' link to identify other cases in the callset for all the genes above. 

It works in a RGP project. Perhaps it is this project only? There's only 1 case in this project so I cannot test other cases. 

The link you shared seems to work for me, but not when I directly select 'Gene Search' or 'seqr'. Let me know what other info would be useful.

**Comment by hanars** [comment]
So that link you provided links to 9 variants, so I can't be sure which variant you were talking about

**Comment by hanars** [comment]
@lynnpais can you please provide more detail about what exactly you are doing to get this issue. Please be specific about what variant(s) you are using to trigger the errors
